Sync `svg/pservers` from WPT Upstream

Ahmad-S792 opened the pull request:
#### 7582df51ec03992d3046e18941f60861e97cf98a
<pre>
Sync `svg/pservers` from WPT Upstream

<a href="https://bugs.webkit.org/show_bug.cgi?id=287399">https://bugs.webkit.org/show_bug.cgi?id=287399</a>
<a href="https://rdar.apple.com/144523376">rdar://144523376</a>

Reviewed by Tim Nguyen.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/df03af416136a50c18501e06fd0cd3ff697da851">https://github.com/web-platform-tests/wpt/commit/df03af416136a50c18501e06fd0cd3ff697da851</a>

* LayoutTests/TestExpectations:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-inheritance-not-in-rendertree-01.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-inheritance-not-in-rendertree-02.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-inheritance-not-in-rendertree-01.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-inheritance-not-in-rendertree-02.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/reference/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/scripted/w3c-import.log:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-basic-002-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-fully-overlapping-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/pservers/reftests/stop-color-currentcolor-dynamic-001-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-basic-002-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-fully-overlapping-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/pservers/reftests/stop-color-currentcolor-dynamic-001-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-basic-002-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-fully-overlapping-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/pservers/reftests/stop-color-currentcolor-dynamic-001-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-basic-002-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-fully-overlapping-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/pservers/reftests/stop-color-currentcolor-dynamic-001-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-currentcolor-1-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-currentcolor-1.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-currentcolor-2-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-currentcolor-2.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-invalid-uri-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-invalid-uri.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-1-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-1.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-2-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-2.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-3-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-3.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-color-interpolation-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-color-interpolation.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-01-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-01.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-02-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-02.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-03-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-03.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-basic-001-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-basic-002-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-basic-003-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-basic-004-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-basic-005-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-bicubic-001-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-complex-001-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-inheritance-template-pattern-removed-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-inheritance-template-pattern-removed.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-opacity-01-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-opacity-01.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-text-01-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-text-01.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-01-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-01.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-02-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-02.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-03-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-03.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-viewbox-01-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-viewbox-01.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-basic-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-fully-overlapping-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/reference/gradient-color-interpolation-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/reference/pattern-opacity-01-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/reference/pattern-transform-01-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/reference/pattern-viewbox-01-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/stop-color-currentcolor-dynamic-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/stroke-fallback-invalid-uri-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/stroke-fallback-invalid-uri.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/scripted/pattern-transform-clear-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/scripted/pattern-transform-clear.svg:

Canonical link: <a href="https://commits.webkit.org/290204@main">https://commits.webkit.org/290204@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/c3b670f9c898c7f70c329650ee95565e3f7b8668

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/89228 "1 style error") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/8752 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/44069 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/94210 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/39989 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/9140 "Built successfully") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/123/builds/17053 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/68746 "Failure limit exceed. At least found 1 new test failure: imported/w3c/web-platform-tests/webvtt/rendering/cues-with-video/processing-model/disable_controls_reposition.html (failure)") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/26416 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/92230 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/7006 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/80988 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/49107 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/6755 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/35369 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/39096 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/77116 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/36356 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/96043 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/16408 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/12096 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/77624 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/16664 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/76775 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/76918 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/18979 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/21329 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/19947 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/9526 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/16422 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/16163 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/19614 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/17944 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/3030535f34c88ed72dddd6ca92441b72fd849c59. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/40352

Ahmad-S792 commented:
`style-check` failures in on PNG files, which were synced from WPT upstream. So I am going to ignore it.

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/ace93b07b946a117876d2efbdb9758521d230abc. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/40352

aproskuryakov commented:
Please add a justification when importing WPT tests. The cost is obvious (slower EWS, missing regressions in the noise from too many tests), so benefits need to be accordingly sizable.

Ahmad-S792 commented:
> Please add a justification when importing WPT tests. The cost is obvious (slower EWS, missing regressions in the noise from too many tests), so benefits need to be accordingly sizable.

@aproskuryakov - I am fixing fallback handling for paint server and rather than adding test locally, I uploaded them on WPT and noticed that the directory was completely outdated so rather than just adding my uploaded tests, I ended up syncing full. PR in question, which I need to rebase post this - https://github.com/WebKit/WebKit/pull/40327

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/c3b670f9c898c7f70c329650ee95565e3f7b8668. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/40352

Ahmad-S792 committed:
Sync `svg/pservers` from WPT Upstream

https://bugs.webkit.org/show_bug.cgi?id=287399
rdar://144523376

Reviewed by Tim Nguyen.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/df03af416136a50c18501e06fd0cd3ff697da851

* LayoutTests/TestExpectations:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-inheritance-not-in-rendertree-01.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-inheritance-not-in-rendertree-02.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-inheritance-not-in-rendertree-01.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-inheritance-not-in-rendertree-02.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/reference/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/scripted/w3c-import.log:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-basic-002-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-fully-overlapping-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/pservers/reftests/stop-color-currentcolor-dynamic-001-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-basic-002-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-fully-overlapping-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/pservers/reftests/stop-color-currentcolor-dynamic-001-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-basic-002-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-fully-overlapping-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/pservers/reftests/stop-color-currentcolor-dynamic-001-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-basic-002-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-fully-overlapping-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/pservers/reftests/stop-color-currentcolor-dynamic-001-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-currentcolor-1-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-currentcolor-1.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-currentcolor-2-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-currentcolor-2.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-invalid-uri-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-invalid-uri.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-1-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-1.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-2-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-2.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-3-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/fill-fallback-none-3.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-color-interpolation-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-color-interpolation.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-01-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-01.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-02-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-02.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-03-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/gradient-transform-03.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-basic-001-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-basic-002-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-basic-003-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-basic-004-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-basic-005-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-bicubic-001-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/meshgradient-complex-001-expected.png:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-inheritance-template-pattern-removed-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-inheritance-template-pattern-removed.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-opacity-01-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-opacity-01.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-text-01-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-text-01.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-01-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-01.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-02-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-02.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-03-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-transform-03.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-viewbox-01-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/pattern-viewbox-01.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-basic-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/radialgradient-fully-overlapping-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/reference/gradient-color-interpolation-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/reference/pattern-opacity-01-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/reference/pattern-transform-01-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/reference/pattern-viewbox-01-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/stop-color-currentcolor-dynamic-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/stroke-fallback-invalid-uri-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/reftests/stroke-fallback-invalid-uri.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/scripted/pattern-transform-clear-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/pservers/scripted/pattern-transform-clear.svg:

Canonical link: https://commits.webkit.org/290204@main

webkit-commit-queue commented:
Committed 290204@main (7582df51ec03): <https://commits.webkit.org/290204@main>

Reviewed commits have been landed. Closing PR #40352 and removing active labels.
